sessions/react-state-2/conditional-usestate/README.md

```diff
@@ -0,0 +1,30 @@
+# React State 2: Conditional useState
+
+If you try to render this challenge in the browser, you won't see the app, but an eslint error regarding the `useState` hook being conditionally rendered.
```

Comment:
I was a bit irritated because the App was rendered in the browser. It wasn't clear the error occurs after clicking the button.

```suggestion
Click the button "Show Message" to render a hidden message to the UI. The App will crash because there is a bug in the code. You won't see the hidden message, but an eslint error regarding the `useState` hook being conditionally rendered.
```